### ISSUE TITLE
bruig: new contacts not displayed in the contact list after KX

### ISSUE THREAD

**Issue by artikozel** [companyzero/bisonrelay#50]
After a KX new contacts do not automatically appear in the contacts sidebar until the app is restarted.

Ubuntu 22.04, bisonrelay-debian-amd64-0.1.1

**Comment by joshlupo** [comment]
I experienced this as well.  I was not connected with chappjc but then someone else, who I was connected with, replied to a post that chappjc did.  On the post page I clicked a button which successfully did a key exchange with chappjc but, after that, chappjc still did not show in the list of people available to chat with, even though I could see his posts and comment on them.  

I then closed the GUI and restarted it.  After restart I was able to see chappjc in the list of people I could chat with.

**Comment by sumiflow** [comment]
I experienced this as well. I was not connected with chappjc but then someone else, who I was connected with, replied to a post that chappjc did. On the post page I clicked a button which successfully did a key exchange with chappjc but, after that, chappjc still did not show in the list of people available to chat with, even though I could see his posts and comment on them.

I then closed the GUI and restarted it. After restart I was able to see chappjc in the list of people I could chat with.
This was on Windows 10
